src/index.html

```diff
@@ -21,7 +24,420 @@
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__links">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__nav-item"
+            data-qa="nav-hover"
+          >
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main page__main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="card-image"
+        />
+
+        <div class="card__info">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-tag">
+          <p class="card__price">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```